group/iab/IAB_Decisions.md

```diff
@@ -54,56 +59,47 @@ for the IAB chair to determine whether it will be held open or secret.
 ## Normal Process
 
 The IAB chair (or executive director on the instructions of the chair) will 
-typically follow the following process:
+typically follow this process to initiate a vote:
 
-- If the request for a vote comes from an external source (e.g., the NomCom
-  provides its IESG slate for confirmation), then, as soon as possible after
-  receiving the request, the IAB chair adds the vote topic to the agenda of the
-  next IAB teleconference. If the next meeting is less than 48 hours away, the
-  chair should instead add the topic to the following teleconference.
+- The chair emails the list of voting members to start an email discussion on the
```

Comment:
s/email discussion/discussion

Reason: I do not think we should limit discussion to email.